### DISCUSSION TITLE
Search: Jump to position of search result instead of document beginning

### DISCUSSION THREAD

**Discussion by klener89** [outline/outline#1549] [General]
Currently found searches bring the user to the beginning of the document. 
Especially on longer documents, I would think, that the expected behaviour after a search result is, to jump directly to the position of the searched pattern in the document. If I read it correctly, the positions of the strings are already tracked in `documents.searchVector`. Is there a functionality to jump to a specific location in the document yet?

**Comment by tommoor** [comment]
This functionality does not exist but it's a great idea – would be very cool to have. There is a standard for this under discussion – unfortunately it's not very far along yet, but it might provide some inspiration for solving:
https://wicg.github.io/scroll-to-text-fragment/

**Reply by klener89** [reply]
Its already implement in the newest version of [chromium](https://blog.chromium.org/2019/12/chrome-80-content-indexing-es-modules.html#:~:text=Text%20URL%20Fragments&text=text,-parameter&text=:~:text=On%20islands,%20birds%20can%20contribute%20as%20much%20as%2060%25%20of%20a%20cat's%20diet), however not on safari&[firefox](https://github.com/mozilla/standards-positions/issues/194).
Its also working working with outline e.g. with [this shared content from you to search analytics](https://inside.getoutline.com/share/d45f8a75-3246-4da3-ad35-c6a0655edbc8##:~:text=No%20immediate%20changes%20to)
But concerning the current status of implementation and the long & messy urls, Im not sure whether this is an option. I feel its rather a feature for websites without a dedicated search functionality. 

(moreover Im sorry, but I dont feel comfortable/skilled enough to contribute in JS at the moment. But im really impressed on what you put together here. It feels like a loved project in every detail)

**Comment by opidopiopi** [comment]
Is there any news to this?
Maybe instead you could highlight the matching string in the document?
Currently, with long documents you have to search a second time to find the match in the resulting document.
